r-package/grf/R/input_utilities.R

```diff
@@ -30,27 +30,34 @@ validate_X <- function(X, allow.na = FALSE) {
   has.missing.values
 }
 
-validate_observations <- function(V, X) {
+validate_observations <- function(V, X, binary = FALSE) {
```

Comment:
If this parameter is only used for the censor currently, I think it's simplest to just move the binary check into `survival_forest` itself.